doc/devices/aer.rst

```diff
@@ -60,25 +56,35 @@ You can change a ``'qiskit.aer'`` device's backend with the ``backend`` argument
 
 .. code-block:: python
 
-    dev = qml.device('qiskit.aer', wires=2, backend='unitary_simulator')
+    dev = qml.device('qiskit.aer', wires=2, backend='aer_simulator_statevector')
 
-Backend options
-~~~~~~~~~~~~~~~
+Backend Methods and Options
+~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Qiskit's backends can take different *backend options*, for example to specify the numerical
-precision of the simulation.
-You can find a list of backend options in the backends' respective API documentations:
+This ``AerSimulator`` backend has several available methods, which
+can be passed via the ``method`` keyword argument. For example
+``'automatica'``, ``'statevector'``, and ``'unitary'``.
```

Comment:
```
This plugin supports the ``'automatic'``, ``'statevector'``, ``'density_matrix'``, and ``'unitary'`` methods.
```

I tested the methods in qiskit documentation.  ``"stabilizer"``, ``"extended_stabilizer"``, ``"matrix_product_state"``, and ``"superop"`` all raise errors and fail.

Reply:
This is a very good catch, I'll update the documentation and validation.